docs/process.md

```diff
@@ -174,6 +174,53 @@ updating `emcc.rst` in a PR, the following should be done:
 See notes above on installing sphinx.
 
 
+## Updating the LLVM libraries
+
+We maintain our ports of compiler-rt, libcxx, libcxxabi, and libunwind under
+https://github.com/emscripten-core/emscripten/tree/main/system/lib from
+[the upstream LLVM repository][llvm_repo] and periodically update them to a newer
+version when a new LLVM release comes out. To update our libraries to a newer
+LLVM release:
+
+1. Create a new branch for the libraries in our LLVM fork
+   (https://github.com/emscripten-core/llvm-project). For example, the branch
+   for LLVM 16 is
+   https://github.com/emscripten-core/llvm-project/tree/emscripten-libs-16. We
+   create a single branch for a major LLVM version, so if you want to update a
+   minor version within the same major version, you can use the same branch.
+1. Create a PR to merge new LLVM release tag in the upstream repo into our
+   library branch. For example, if we want to merge LLVM 16.0.6 release into our
+   `emscripten-libs-16` branch, you can do
+   ```
+   git co emscripten-libs-16
+   git remote add upstream https://github.com/llvm/llvm-project
+   git fetch --tags upstream
+   git merge llvmorg-16.0.6
+   ```
+   An example of such PR is emscripten-core/llvm-project#3.
+1. Add downstream changes from emscripten to the updated library branch in LLVM
+   fork. You can use [`push_llvm_changes.py`][push_llvm_changes_emscripten] to
+   do that.
+   ```
+   ./system/lib/push_llvm_changes.py <Emscripten's LLVM fork directory>
+   ```
+   (The library branch should be checked out in your Emscripten's LLVM fork
+   directory.)
+   Example of such PR is emscripten-core/llvm-project#5.
```

Comment:
Fixed it. I followed the wrong order while I was updating libraries last week but it ended up fine just because I was updating 16.0.4->16.0.6 so there was no changes to the libraries I guess.